File: python/OperacoesBasicas.py
class OperacoesBasicas:
    def soma(self, a, b):
        return a + b

    def a_potencia_b(self, a, b):
        a_potencia_b = a**b
        return a_potencia_b

# Crie uma instância da classe
operacoes_basicas = OperacoesBasicas()
# Chame os métodos usando a instância
resultado_soma = operacoes_basicas.soma(4, 5)
resultado_potencia = operacoes_basicas.a_potencia_b(4, 4)

print(resultado_soma)  # Irá imprimir 9
print(resultado_potencia)  # Irá imprimir 16

File: java/OperacoesBasicas.java
public class OperacoesBasicas {
    public static int Soma (int a, int b) {
        return a + b;
    }

    public static int APotenciaB (int a, int b) {
        int APotenciaB = (int) Math.pow(a, b);
        return APotenciaB;
    }
    public static void main (String[] args) {
        int retorno = Soma(50, 0);
        int APotenciaB = APotenciaB(4,5);
        System.out.println("O valor de a mais b eh: " + retorno);
        System.out.println("O valor de a potencia b eh: " + APotenciaB);
    }
}
